module/proto/gravity/v1/msgs.proto

```diff
@@ -12,33 +12,46 @@ option go_package = "github.com/peggyjv/gravity-bridge/module/x/gravity/types";
 
 // Msg defines the state transitions possible within gravity
 service Msg {
+  rpc SendToEVM(MsgSendToEVM) returns (MsgSendToEVMResponse) {}
+  rpc CancelSendToEVM(MsgCancelSendToEVM) returns (MsgCancelSendToEVMResponse) {
+  }
+  rpc RequestEVMBatchTx(MsgRequestEVMBatchTx)
+      returns (MsgRequestEVMBatchTxResponse) {}
+  rpc SubmitEVMTxConfirmation(MsgSubmitEVMTxConfirmation)
+      returns (MsgSubmitEVMTxConfirmationResponse) {}
+  rpc SubmitEVMEvent(MsgSubmitEVMEvent) returns (MsgSubmitEVMEventResponse) {}
+  rpc SetDelegateChainKeys(MsgDelegateChainKeys)
+      returns (MsgDelegateChainKeysResponse) {}
+
+  // deprecated endpoints
   rpc SendToEthereum(MsgSendToEthereum) returns (MsgSendToEthereumResponse) {
-    // option (google.api.http).post = "/gravity/v1/send_to_ethereum";
+    option deprecated = true;
   }
   rpc CancelSendToEthereum(MsgCancelSendToEthereum)
       returns (MsgCancelSendToEthereumResponse) {
-    // option (google.api.http).post = "/gravity/v1/send_to_ethereum/cancel";
+    option deprecated = true;
   }
   rpc RequestBatchTx(MsgRequestBatchTx) returns (MsgRequestBatchTxResponse) {
-    // option (google.api.http).post = "/gravity/v1/batchtx/request";
+    option deprecated = true;
   }
   rpc SubmitEthereumTxConfirmation(MsgSubmitEthereumTxConfirmation)
       returns (MsgSubmitEthereumTxConfirmationResponse) {
-    // option (google.api.http).post = "/gravity/v1/ethereum_signature";
+    option deprecated = true;
```

Comment:
Why `deprecated = true`?

Reply:
Because the one you should be using now is SubmitEVMTxConfirmation, which includes chain ID in its input message. Leaving this in place but deprecated so that users aren't forced to update. It will redirect in the background to use SubmitEVMTxConfirmation with chain ID 1